csharp/Svix/Webhook.cs

```diff
@@ -31,6 +31,16 @@ public Webhook(string key)
             this.key = Convert.FromBase64String(key);
```

Comment:
JS is not one of these languages, but quite a few languages have an easy `as_bytes()`, I wonder if we should just have a raw that accepts bytes and not have a string variant for literally all of the languages other than JS?